app/assets/stylesheets/help.scss

```diff
@@ -391,7 +391,7 @@ html, body {
   .card.support {
     display: none;
 
-    &.IN {
+    &.IN, &.BD, &.ET {
```

Comment:
Support card not showing in BD is intentional. We don't have a Whatsapp point of contact in BD, and a lot of people don't use Whatsapp.

Good catch with ET though. Perhaps we should flip these CSS rules to _hide_ for `&.BD`, instead of _showing_ for countries.

Reply:
Thanks @harimohanraj89 good point — I'll update to hide in BD only :)